rpc/core/src/api/ops.rs

```diff
@@ -86,6 +86,8 @@ pub enum RpcApiOps {
     GetCoinSupply,
     /// Get DAA Score timestamp estimate
     GetDaaScoreTimestampEstimate,
+    /// Get UTXO Return Addresses
+    GetUtxoReturnAddress,
```

Comment:
@coderofstuff  Please relocate the GetUtxoReturnAddress at the end of the enum as this currently breaks borsh serialization.

Reply:
Fixed in [652ae15](https://github.com/kaspanet/rusty-kaspa/pull/436/commits/652ae15a84262660f8bfef8119073a1aa7869b87)